test/integ/hostOnlyAPI/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+#
+# Copyright 2022 Andrea Bocci
+#
+# This file is part of alpaka.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+set(_TARGET_NAME "hostOnlyAPITest")
+
+append_recursive_files_add_to_src_group("src/" "src/" "cpp" _FILES_SOURCE)
+
+add_executable(
+    ${_TARGET_NAME}
+    ${_FILES_SOURCE})
+target_link_libraries(
+    ${_TARGET_NAME}
+    PRIVATE common)
+
+target_compile_definitions(${_TARGET_NAME} PRIVATE ALPAKA_HOST_ONLY ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED)
```

The `ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED` shouldn't be necessary here, it is automatically added by the top-level `CMakeLists.txt`.

```suggestion
target_compile_definitions(${_TARGET_NAME} PRIVATE ALPAKA_HOST_ONLY)
```

---

I did try without it, but it failed in the CI on the Windows NVCC targets.

---

Hm, it seems that the serial back-end is explicitly disabled for the Windows-Release runners. I'll look into this.

---

I've opened #1577 to run the tests without this change.

---

OK, as expected #1577 failed few of the tests:
  - [linux_nvcc-11.4_clang-11_release](https://gitlab.com/hzdr/crp/alpaka/-/jobs/1991403986)
  - [windows_nvcc-11.2_cl-2019_release_cuda-only_c++17](https://github.com/alpaka-group/alpaka/runs/4867897549?check_suite_focus=true) and similar